src/test/lib/ApiTest.cpp

```diff
@@ -4227,6 +4227,109 @@ void QuicTest_QUIC_PARAM_CONN_STATISTICS_V2_PLAT(MsQuicRegistration& Registratio
     }
 }
 
+
```

Comment:
Let's add a test for a NULL buffer to query the length.

Reply:
For sure!